crates/oxc_transformer/src/typescript/module.rs

```diff
@@ -7,12 +8,13 @@ use super::diagnostics;
 use crate::TransformCtx;
 
 pub struct TypeScriptModule<'a, 'ctx> {
+    only_remove_type_imports: bool,
```

Comment:
can you reference https://babeljs.io/docs/babel-preset-typescript#onlyremovetypeimports